packages/rest/src/rest.ts

```diff
@@ -226,3 +226,31 @@ setDecoratorNamespace(
   $listsResource,
   $action
 );
+
+export interface Example {
+  pathOrUri: string;
+  title: string;
+}
+
+const exampleKey = Symbol();
+export function $example(program: Program, entity: Type, pathOrUri: string, title: string) {
+  if (entity.kind !== "Operation") {
```

Comment:
I think we might have discussed this in the teams channel but this should go in `cadl-autroest` not the `rest` library

Reply:
makes sense! will do.